dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/internal/OidcClientImpl.java

```diff
@@ -430,6 +432,28 @@ public ProviderAuthenticationResult authenticate(HttpServletRequest req,
         return oidcClientAuthenticator.authenticate(req, res, oidcClientConfig);
     }
 
+    @Override
+    public void logoutIfSessionInactive(HttpServletRequest req, String provider) {
+        OidcClientConfig oidcClientConfig = oidcClientConfigRef.getService(provider);
+        if (!oidcClientConfig.isBackchannelLogoutSupported()) {
+            return;
+        }
+
+        HttpSessionCache httpSessionCache = oidcClientConfig.getHttpSessionCache();
+        String httpSessionId = req.getSession().getId();
```

Comment:
This might need to be something like:
```suggestion
        HttpSession httpSession = req.getSession(false);
        String httpSessionId = (httpSession == null) ? null : httpSession.getId();
```

Reply:
changed it so it doesn't try to create a session if one doesn't already exist.